assets/wizards/advertising/views/ad-units/index.js

```diff
@@ -167,6 +162,70 @@ const AdUnits = ( {
 					isSuccess
 				/>
 			) }
+			<div className="flex items-end" style={ { margin: '-30px 0' } }>
+				<TextControl
+					className="mr2"
+					label={ __( 'Network code', 'newspack' ) }
+					value={ networkCode }
+					onChange={ setNetworkCode }
+					disabled={ serviceData.status.connected }
+				/>
+				<Card buttonsCard noBorder>
+					<Button onClick={ saveNetworkCode } isPrimary disabled={ serviceData.status.connected }>
```

Comment:
If you've uploaded credentials, maybe there's no reason to even show the disabled "Save" button? I would prefer to hide it in this case.